admin/app/components/solidus_admin/ui/table/component.html.erb

```diff
@@ -10,81 +10,82 @@
 >
   <% toolbar_classes = "h-14 p-2 bg-white border-b border-gray-100 justify-start items-center gap-2 visible:flex hidden:hidden" %>
 
-  <div class="<%= toolbar_classes %>" data-<%= stimulus_id %>-target="searchToolbar">
-    <%= form_with(
-      url: @search_url,
-      method: :get,
-      html: {
-        id: search_form_id,
-        class: 'flex-grow',
-        "data-turbo-frame": table_frame_id,
-        "data-turbo-action": "replace",
-        "data-#{stimulus_id}-target": "searchForm",
-        "data-action": "reset->#{stimulus_id}#search",
-      },
-    ) do |form| %>
-      <label class="items-center gap-1 p-0 inline-flex w-full justify-start relative">
-        <%= render @icon_component.new(name: 'search-line', class: "w-[1.4em] h-[1.4em] fill-gray-500 absolute ml-3") %>
-        <span class="sr-only">
-          <%= t('.search_placeholder', resources: resource_plural_name) %>
-        </span>
-        <input
-          name="q[<%= @search_key %>]"
-          value="<%= params.dig(:q, @search_key) %>"
-          type="text"
-          placeholder="<%= t('.search_placeholder', resources: resource_plural_name) %>"
-          class="peer w-full placeholder:text-gray-400 py-1.5 px-10 bg-white rounded border border-gray-300"
-          data-<%= stimulus_id %>-target="searchField"
-          data-action="<%= stimulus_id %>#search"
-        >
-        <button
-          class="absolute right-0 mr-3 peer-placeholder-shown:hidden"
-          data-action="<%= stimulus_id %>#clearSearch"
-        >
-          <%= render @icon_component.new(name: 'close-circle-fill', class: "w-[1.4em] h-[1.4em] fill-gray-500") %>
-          <span class="sr-only"><%= t('.clear') %></span>
-        </button>
-      </label>
-    <% end %>
-
-    <div class="ml-4">
-      <%= render @button_component.new(
-        text: t('.cancel'),
-        scheme: :ghost,
-        "data-action": "#{stimulus_id}#cancelSearch",
-      ) %>
-    </div>
-  </div>
-
-  <% if @filters.any? %>
-    <div class="<%= toolbar_classes %>" data-<%= stimulus_id %>-target="filterToolbar">
-      <div class="font-semibold text-gray-700 text-sm px-2"><%= t('.refine_search') %>:</div>
-      <% @filters.each do |filter| %>
-        <label class="flex gap-2 px-2">
-          <%= render component('ui/forms/checkbox').new(
-            name: filter[:name],
-            value: filter[:value],
-            size: :s,
-            form: search_form_id,
-            'data-action': "#{stimulus_id}#search",
-          ) %>
-          <span class="text-gray-700 leading-none text-sm self-center"><%= filter[:label] %></span>
+  <div role="search">
```

Comment:
The form is only part of the search area. For instance, we also have the scopes and filters.

Reply:
Although it would be technically correct, I feel it would abuse semantics too much. Let's use the `div` for now, and we can revisit once all the filters are in place.